src/allowed-blocks-setup.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { map, without } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { getBlockTypes } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { JETPACK_DATA_PATH } from '../jetpack/projects/plugins/jetpack/extensions/shared/get-jetpack-data';
+import { registerBlock as registerJetpackLayoutGridBlock } from '../block-experiments/blocks/layout-grid/src';
+import {
+	registerContactInfoBlock as registerJetpackContactInfoBlock,
+	registerStoryBlock as registerJetpackStoryBlock,
+} from '../jetpack/projects/plugins/jetpack/extensions/editor.native';
+
+// Please also consider updating ./block-support/supported-blocks.json
+const availableJetpackBlocks = {
+	'contact-info': { available: true },
+	'layout-grid': { available: true },
+	story: { available: true },
+};
+
+const setJetpackData = ( {
+	isJetpackActive = false,
+	userData: tracksUserData = null,
+	siteFragment = null,
+	blogId: wpcomBlogId = 1,
```

Comment:
We sort of did have it just not in this way where it would be used if the value provided was `undefined`. We had a default for the entire object, i.e., `{ blogId: 1, isJetpackActive: true }`. I agree that it doesn't make practical sense for it to be `1` as the default though. Would `0` make more sense as a default falsey value? Although there may legitimately be a blog with `id` of `0`. So maybe `NaN`? I'm not particularly fond of simply leaving it `undefined`, but that might actually make practical sense as well. WDYT?

- [ ] Adjust the default value of `blogId` to not be `1`.

Reply:
Right, I see that we were setting this default value here:
https://github.com/wordpress-mobile/gutenberg-mobile/blob/6fea169a3c04e6fa441c5369d60539df2a407ca9/src/index.js#L25

To be honest I don't have enough context on why we use `1` as the default blog id but probably it won't produce any undesired side-effect if we keep it. I found that this was introduced in [this PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1934) long ago but it doesn't provide any clue about the value.

On the Gutenberg repository, when the editor is initialized we use also the value `1` as the default value for the post id so maybe this value is the standard default value for these type of ids:
https://github.com/WordPress/gutenberg/blob/0ebe9b3a9183a907d6eaf26c3f9f1860bd29dd2e/packages/react-native-editor/src/index.js#L48

Reply:
Since we're already setting the default value to `1` [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/6fea169a3c04e6fa441c5369d60539df2a407ca9/src/index.js#L25), I think it's ok to keep it so I'm resolving this conversion 👍 .

Reply:
Yeah, that's basically what I gathered as well when reading through the code. It honestly didn't seem intuitive to me that the default value be something like `1` as I would assume `1` could literally be the second post ever made. Something like `NaN` would probably provide a better cue, but we are defaulting to `1` if the `post` is not passed as a `prop` to the `Editor` component. 🤷 

Thanks for sending the links over.